EngineLayer/spectralLibrarySearch/spectralLibraryReader.cs

```diff
@@ -0,0 +1,112 @@
+﻿using Proteomics.Fragmentation;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace EngineLayer.SpectralLibrarySearch
+{
+    public static class SpectralLibraryReader
+    {
+
+        public static Dictionary<string, SpectralBook> ReadSpectralLibrary(string filePath)
```

Comment:
Please comment here: Describe what type of files this function can read. Where can people get this type of file.